client/app/(auth)/api/accountApi.ts

```diff
@@ -0,0 +1,11 @@
+import apiClient from '@/_api/base'
+import { UserType } from '@/_types'
+import { AccountSchema } from '../schema'
+
+export const accountApi = async (data: AccountSchema) => {
```

Comment:
3 things I want you to modify!

1. accountApi is not auth feature, so could you move this file to `/app/account/api` directory ?
2. More specifically, this is account edit api, so could you rename to `accountUpdateApi` ?
3. You need to change `post` to `put` method

Reply:
I fixed it.
77101ac8c1a9c6e6d1bd8d41a585c8e468692b77

Reply:
I think you forgot No.3!